usagov/vote-gov-drupal
# Feature/vote 244: Add compound field - NVRF Field

## Jira ticket

[VOTE-244](https://cm-jira.usa.gov/browse/VOTE-244)

## Description

Create a new custom module which provides a compound field called NVRF Field which is added to the State/Territory Page content type.

## Deployment and testing

### Pre-deploy

1. None

### Post-deploy

1. Locally, run `lando drush cim` to import changes.

### QA/Test

1. Login with an account with Admin role
2. Goto the Content Admin page
3. Locate a pre-existing State/Territory content item and Edit it
4. Once the edit page loads scroll down towards the bottom and confirm that the "NVRF Fields" compound field is available
5. Confirm that you are able to select a NNRF Field term using the autocomplete field
6. Confirm that you can add multiple entries
7. Save the form
8. Revisit the form and confirm that the values are saved and visible

## Checklist for the Developer




## Checklist for the Peer Reviewers
